### PR TITLE
chore: disable renovate for unit test projects

### PR DESCRIPTION
**Pull request by jbedard**
I think this is just causing noise and other then security updates these aren't worth updating.

---

### Test plan

- Covered by existing test cases

### CODE REVIEW

File: npm/extensions.bzl [hunk 1]
```diff
@@ -11,8 +11,8 @@ load("//npm/private:npm_translate_lock.bzl", "npm_translate_lock_lib", "npm_tran
 load("//npm/private:npm_translate_lock_helpers.bzl", npm_translate_lock_helpers = "helpers")
 load("//npm/private:npm_translate_lock_macro_helpers.bzl", macro_helpers = "helpers")
 load("//npm/private:npm_translate_lock_state.bzl", "npm_translate_lock_state")
-load("//npm/private:pnpm_extension.bzl", "DEFAULT_PNPM_REPO_NAME", "resolve_pnpm_repositories")
```
Comment (jbedard) [inline npm/extensions.bzl:14]: These are unrelated and caused by https://github.com/aspect-build/rules_js/pull/1933 being merged before rebasing to get a buildifier upgrade.

**Review by DavidZbarsky-at — APPROVED** [review]

**Review by dzbarsky — APPROVED** [review]
